### ISSUE TITLE
Generated typescript project with sass throws error.

### ISSUE THREAD

**Issue by TheRealSyler** [webpack/webpack-cli#3171]
**What is the current behavior?**

crashes on `yarn serve`, the reason is in the generated webpack config file there is an undefined variable called `stylesHandler`.

**To Reproduce**

1. In an empty folder run `yarn add -D @webpack-cli/generators && npx webpack-cli init`

2. Choose these options: 
```sh
? Which of the following JS solutions do you want to use? Typescript
? Do you want to use webpack-dev-server? Yes
? Do you want to simplify the creation of HTML files for your bundle? Yes
? Do you want to add PWA support? Yes
? Which of the following CSS solutions do you want to use? SASS
? Will you be using CSS styles along with SASS in your project? No
? Will you be using PostCSS in your project? No
? Do you want to extract CSS for every file? No
? Do you like to install prettier to format generated configuration? Yes
? Pick a package manager: yarn
```

3. run `yarn serve`

**Expected behavior**

Not crash when running `yarn serve`.

**Please paste the results of `npx webpack-cli info` here, and mention other relevant information**
```sh
  System:
    OS: Windows 10 10.0.22000
    CPU: (6) x64 Intel(R) Core(TM) i5-8600K CPU @ 3.60GHz
    Memory: 8.29 GB / 15.96 GB
  Binaries:
    Node: 16.14.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.15 - C:\Program Files\nodejs\yarn.CMD
    npm: 8.5.4 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22000.120.0), Chromium (99.0.1150.39) 
    Internet Explorer: 11.0.22000.120
  Packages:
    html-webpack-plugin: ^5.5.0 => 5.5.0
    sass-loader: ^12.6.0 => 12.6.0
    ts-loader: ^9.2.8 => 9.2.8
    webpack: ^5.70.0 => 5.70.0
    webpack-cli: ^4.9.2 => 4.9.2
    webpack-dev-server: ^4.7.4 => 4.7.4
    workbox-webpack-plugin: ^6.5.1 => 6.5.1
```

**Comment by TheRealSyler** [comment]
ill take a look at it myself so i might create a pr later.

**Comment by TheRealSyler** [comment]
the service worker file also doesn't seem to be created.

**Comment by TheRealSyler** [comment]
fixed with  #3412
